executor.c

```diff
@@ -0,0 +1,90 @@
+//
+//  executor.c
+//  connectivi_graph
+//
+//  Created by Artem on 26/08/2022.
+//  Copyright © 2022 Artem. All rights reserved.
+//
+
+#include "finding_connectivity_components.h"
+
+
+Pair_int_int* read_grath_from_txt(char* file_path_name, int* points_cnt, int* arc_cnt){
+    Pair_int_int *v;
+    
+    FILE *in_file;
+    in_file = fopen(file_path_name, "r");
+    fscanf(in_file, "points count = %d", points_cnt);
+    fscanf(in_file, "\narcs count = %d", arc_cnt);
+    v = (Pair_int_int *)(calloc(*arc_cnt, sizeof(Pair_int_int)));
+    // graph is represented as a list of edges, which is necessary for Prim algorithm
+    // and significantly reduces the amount of stored data
+    for(int i = 0; i < *arc_cnt; ++i)
+        fscanf(in_file, "%d %d", &v[i].first, &v[i].second);
+    fclose(in_file);
+    return v;
+}
+
+
+int find_root(Tree_point *u_mass, Tree_point u){
+    while (u.me != u.my_root)
+        u = u_mass[u.my_root];
+    return u.me;
+}
+
+void create_point(Tree_point *u, int i){
```

Comment:
This function not actually creates object but initialize it. Some kind of `init*` prefix may be clearer.

Reply:
Agree, it will make code more clear